Update creating_documents.rst (#696)
sulu-io/sulu-docs: reference/components/document-manager/creating_documents.rst

In order for the document manager to recognize existing managed documents and
persist new ones, you must add some mapping to your configuration.

> This configuration can be done in you applications ``config/packages/sulu_document_manager.yaml`` file:

Author: ```suggestion
The following configuration should be added to the ``config/packages/sulu_document_manager.yaml`` file:
```
